src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -372,6 +379,40 @@ private static void verifyConfig(Configuration testConfig, Configuration parsedC
         }
     }
 
+    /**
+     * Performs verification of violation lines.
+     *
+     * @param config parsed config.
+     * @param file file path.
+     * @param inputConfiguration InputConfiguration object.
+     * @throws Exception if exception occurs during verification process.
+     */
+    private void verifyViolations(Configuration config,
+                                  String file,
+                                  InputConfiguration inputConfiguration) throws Exception {
+        final List<File> files = Collections.singletonList(new File(file));
+        final Checker checker = createChecker(config);
+        final int errs = checker.process(files);
+        final Map<String, List<String>> actualViolations = getActualViolations(errs);
+        checker.destroy();
+        List<Integer> expectedViolationLines;
+        try {
```

Comment:
do we have the case, where exception is thrown?

Reply:
It was the case when the key ```file``` is not in the map.
I refactored it.